README.md

```diff
@@ -33,6 +33,21 @@ Follow these steps to integrate the FULLHAUS PHP-CS-Fixer into your project:
     return $config;
     ```
 
+   
+   > **NOTE:** For TYPO3 projects, adjust the directory to `$config->getFinder()->in(__DIR__ . '/packages');`.
+
+4. Add the following snippet to your root `composer.json`:
+   ```json
+   "scripts": {
+      "fix:php": [
+         "@fix:php:cs"
+      ],
+      "fix:php:cs": "php-cs-fixer fix --config .php-cs-fixer.php"
+   },
+   ```
+
+> ATTENTION: Beware of copy🍝! There might be an existing `scripts` section in your composer.json already. In this case copy the cs-fixer scripts **only**, if not in there, yet. Anyways, IDE will complain about duplicate associative keys.
+
```

Comment:
Indentation should be 4 spaces instead of 3, in order to avoid rendering issues. Furthermore, I would also indent the "Attention" note so it gets aligned with the content above it.

Reply:
Aight you're right with the spaces, then its just the indent on Attention @david-bonhagen